compiler/compiler.cpp

```diff
@@ -237,6 +238,7 @@ bool compiler_execute(CompilerSettings *settings) {
     >> PipeC<SplitSwitchF>{}
     >> PipeC<CollectRequiredAndClassesF>{} >> use_nth_output_tag<0>{}
     >> SyncC<CheckRequires>{}
+    >> PassC<ConvertSpreadOperatorPass>{}
```

Comment:
Do we really need a separate pass for this? Can't this be done in GenTreePostprocess or right in gentree itself?

Reply:
Yep, moved to `GenTreePostprocessPass`.

Reply:
It doesn't seem to work, since we need to get the function `array_merge_spread`, so we need to do the conversion after the functions are collected.